docs/02-app/02-api-reference/04-functions/revalidatePath.mdx

```diff
@@ -8,16 +8,20 @@ description: API Reference for the revalidatePath function.
 > **Good to know**:
 >
 > - `revalidatePath` is available in both [Node.js and Edge runtimes](/docs/app/building-your-application/rendering/edge-and-nodejs-runtimes).
-> - `revalidatePath` will revalidate _all segments_ under a dynamic route segment. For example, if you have a dynamic segment `/product/[id]` and you call `revalidatePath('/product/[id]')`, then all segments under `/product/[id]` will be revalidated as requested.
-> - `revalidatePath` only invalidates the cache when the path is next visited. This means calling `revalidatePath` with a dynamic route segment will not immediately trigger many revalidations at once. The invalidation only happens when the path is next visited.
+> - `revalidatePath` can invalidate by:
```

Comment:
This is probably now worth breaking out of good to know, and into it's own section. I'd like to avoid nested bullets. 

"Examples" maybe. Or "Invalidating Data".

Reply:
Good point broke this out into separate examples.